File: 1970-sorting-the-sentence/sorting-the-sentence.py
class Solution(object):
    def sortSentence(self, s):
        """
        :type s: str
        :rtype: str
        """
        x= s.split()
        x.sort(key=lambda word:int(word[-1]))
        result=[word[:-1]for word in x]
        n=(" ".join(result))
        return n

File: 1970-sorting-the-sentence/sorting-the-sentence.js
/**
 * @param {string} s
 * @return {string}
 */
var sortSentence = function(s) {
    let words=s.split(" ");
    let result=[]
    for(let word of words){
        let pos=word[word.length-1];
        let cleanword=word.slice(0,-1);
        result[pos-1]=cleanword;
    }
    return result.join(" ")
};